src/Arcus.Testing.Tests.Integration/Integration/DataFactory/Fixture/TemporaryDataFactoryDataFlow.cs

```diff
@@ -43,48 +44,75 @@ public class TemporaryDataFactoryDataFlow : IAsyncDisposable
         private TemporaryDataFactoryDataFlow(DataFlowDataType dataType, TestConfig config, ILogger logger)
         {
             _dataType = dataType;
-            _sinkDatasetName = RandomizeWith("sink");
             _linkedServiceName = RandomizeWith("storage");
-            
+
             _arm = new ArmClient(new DefaultAzureCredential());
             _config = config;
             _logger = logger;
 
             Name = RandomizeWith("dataFlow");
-            SourceName = RandomizeWith("source");
+            SourceName = RandomizeWith("sourceName");
+            SinkName = RandomizeWith("sinkName");
+            SinkDataSetName = RandomizeWith("sinkDataSet");
+            SourceDataSetName = RandomizeWith("sourceDataSet");
+            SinkDataSetParameterValue = RandomizeWith("sinkDataSetParameterValue");
         }
 
         private string SubscriptionId => _config["Arcus:SubscriptionId"];
         private string ResourceGroupName => _config["Arcus:ResourceGroup:Name"];
         private DataFactoryConfig DataFactory => _config.GetDataFactory();
         private StorageAccount StorageAccount => _config.GetStorageAccount();
-        private string SourceName { get; }
 
         /// <summary>
         /// Gets the unique name of the temporary DataFlow in Azure DataFactory.
         /// </summary>
         public string Name { get; }
 
+        /// <summary>
+        /// Gets the unique name of the source of the temporary DataFlow in Azure DataFactory.
+        /// This name is the name of the source "step" of the DataFlow, not to be mistaken with the source DataSet name.
+        /// </summary>
+        public string SourceName { get; }
+
         /// <summary>
         /// Gets the unique name of the sink of the temporary DataFlow in Azure DataFactory.
+        /// This name is the name of the sink "step" of the DataFlow, not to be mistaken with the sink DataSet name.
         /// </summary>
-        public string SinkName { get; } = "dataflowsink";
+        public string SinkName { get; }
+
+        /// <summary>
+        /// Gets the unique name of the source DataSet of the temporary DataFlow in Azure DataFactory.
+        /// </summary>
+        public string SourceDataSetName { get; }
+
+        /// <summary>
+        /// Gets the unique name of the source DataSet of the temporary DataFlow in Azure DataFactory.
+        /// </summary>
+        public string SinkDataSetName { get; }
+
+        /// <summary>
+        /// Gets the unique name of the parameter value of the sink DataSet of the temporary DataFlow in Azure DataFactory.
+        /// </summary>
+        public string SinkDataSetParameterValue { get; }
```

Comment:
This can now be deleted?

Reply:
Fixed in [#b0c4faf](https://github.com/arcus-azure/arcus.testing/pull/228/commits/b0c4faf4dc9bc205cb6af8c232fc2a03096d79a4)